examples/behaviors/basic/actions/append/once/style.xml.njk

```diff
@@ -0,0 +1,10 @@
+<style
```

Comment:
is this needed? the button style should already be defined in the macro right?

Reply:
i'm tweaking the button style here. flexdirection = column

Reply:
I think this property should probably be part of the root button style anyway?